examples/poker/server/impl.ts

```diff
@@ -124,20 +129,18 @@ export class Impl implements Methods<InternalState> {
     return Response.ok();
   }
   getUserState(state: InternalState, userId: UserId): PlayerState {
-    const showdown =
-      filterPlayers(state.players, PlayerStatus.WAITING).length === 0 &&
-      filterPlayers(state.players, PlayerStatus.PLAYED).length > 1;
     return {
-      players: state.players.map((player) => {
-        const shouldReveal = player.id === userId || (showdown && player.status === PlayerStatus.PLAYED);
+      players: state.players?.map((player) => {
```

Comment:
why the `?`?

Reply:
Not honestly sure since we don't need it